### PR TITLE
fixes related to go modules migration of gitbase

### PR DESCRIPTION
**Pull request by lwsanty**
previously gitbase dependencies were acquired using command:
```shell
go get -t ./...
```
as soon as gitbase switched to go modules this command fails so to get dependencies we need to use command:
```shell
go mod download
```

Signed-off-by: lwsanty <lwsanty@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.exe
+.idea
```
Comment (erizocosmico) [inline .gitignore]: this should go in your global gitignore, instead of in specific project ones

File: .travis.yml [hunk 2]
```diff
@@ -37,8 +37,9 @@ install:
   - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-6 90
   - sudo service mysql stop
 
+# TODO(lwsanty): we probably need to include go mod case into our CI on 'make dependencies' command to avoid this fixes in the future
 script:
-  - make dependencies
+  - go mod download
```
Comment (erizocosmico) [inline .travis.yml]: instead of that, just adding `GO111MODULE=on` environment variable makes everything else work seamlessly.

File: gitbase.go [hunk 3]
```diff
@@ -11,10 +11,15 @@ func NewToolGitbase() regression.Tool {
 		GitURL:      "https://github.com/src-d/gitbase",
 		ProjectPath: "github.com/src-d/gitbase",
 		BuildSteps: []regression.BuildStep{
+			{
```
Comment (erizocosmico) [inline gitbase.go:14]: this build step is unnecessary if `GO111MODULE=on` is present (see previous comment in travis.yml)

**Review by erizocosmico — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (lwsanty) [reply .travis.yml]: Done

File: .gitignore (hunk 1 above)
Comment (lwsanty) [reply .gitignore]: Done

File: gitbase.go (hunk 3 above)
Comment (lwsanty) [reply gitbase.go:14]: Done

**Review by erizocosmico — APPROVED** [review]

**Review by ajnavarro — APPROVED** [review]

**Review by erizocosmico — APPROVED** [review]

**Review by jfontan — APPROVED** [review]

File: .travis.yml [hunk 4]
```diff
@@ -10,6 +10,7 @@ service:
 
 env:
   - GO111MODULE=on
```
Comment (jfontan) [inline .travis.yml]: There should be only one line:

```
- GO111MODULE=on GOPROXY=https://proxy.golang.org
```

https://docs.travis-ci.com/user/environment-variables/#defining-multiple-variables-per-item

**Review by jfontan — CHANGES_REQUESTED** [review]

**Review by ajnavarro — APPROVED** [review]

**Review by jfontan — APPROVED** [review]

**Pull request merged**